### PR TITLE
Disabled users don't appear in statistics city/state and order statistics

### PR DESCRIPTION
**Pull request by CoopTang**
- Extensions 58/59 Users that are disabled do not show up on statistics regarding orders and city/state
- This affects `Merchant#distinct_cities` and `Item#five_most/least_popular_items`

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -32,10 +32,30 @@ def no_orders?
...
+    find_by_sql("select items.name, sum(quantity) as purchase_amount from items 
+      inner join item_orders on item_orders.item_id = items.id 
+      inner join orders on orders.id = item_orders.order_id 
+      inner join users on users.id = orders.user_id 
+      where \"users\".\"enabled?\" = 't' 
+      group by items.name 
+      order by purchase_amount desc 
+      limit 5;"
+    )
+    # includes(item_orders: [orders: :users]).group(:id).where('user.enabled? = true').select('item.name, sum(quantity) as purchase_amount').order('purchase_amount desc').limit(5)
```
Comment (lrs8810) [inline app/models/item.rb]: @CoopTang Are the comments for us to reference during our evaluation or were they left in by mistake?

Comment (CoopTang) [reply app/models/item.rb]: They were left by mistake while I was trying to get it to work.

Comment (freeheeling) [reply app/models/item.rb]: I took the liberty of deleting the comments.

**Comment by freeheeling** [comment]
Approving for merge.

**Pull request merged**